Detectors/Passive/src/Pipe.cxx

```diff
@@ -8,99 +8,2921 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-/********************************************************************************
- *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
- *                                                                              *
- *              This software is distributed under the terms of the             *
- *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
- *                  copied verbatim in the file "LICENSE"                       *
- ********************************************************************************/
-
-// -------------------------------------------------------------------------
-// -----                    Pipe  file                               -----
-// -----                Created by M. Al-Turany  June 2014             -----
-// -------------------------------------------------------------------------
-
 #include "DetectorsPassive/Pipe.h"
-#include "TGeoManager.h"   // for TGeoManager, gGeoManager
-#include "TGeoMaterial.h"  // for TGeoMaterial
-#include "TGeoMedium.h"    // for TGeoMedium
-#include "TGeoPcon.h"      // for TGeoPcon
-#include "TGeoVolume.h"    // for TGeoVolume
-
-using namespace o2::Passive;
-
-Pipe::~Pipe()
-= default;
-Pipe::Pipe()
-  : FairModule()
-{
-}
+#include <DetectorsBase/Detector.h>
+#include <TGeoCompositeShape.h>
+#include <TGeoCone.h>
+#include <TGeoPcon.h>
+#include <TGeoTorus.h>
+#include <TGeoTube.h>
+#include <TVirtualMC.h>
+#include "TGeoManager.h"  // for TGeoManager, gGeoManager
+#include "TGeoMaterial.h" // for TGeoMaterial
+#include "TGeoMedium.h"   // for TGeoMedium
+#include "TGeoVolume.h"   // for TGeoVolume
+#ifdef NDEBUG
+#undef NDEBUG
```

Comment:
Why is this necessary?

Reply:
To force assert being available in any case.